testing/dart/channel_buffers_test.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 // @dart = 2.6
+
```

Comment:
We currently do not run these tests against the web version and instead maintain separate tests. Can you please copy this test into lib/web_ui/test?

/cc @nturgut how feasible is it to run these tests against the web version? One difference between these tests and ours is that these tests input `dart:ui`, while web tests import `package:ui/ui.dart`.

Reply:
I had actually already ported the tests but forgot to `git add` them, oops.